### ISSUE TITLE
A11y_Model Builder_Image Classification_Environemnt Local GPU_Focus Order: Focus order is not correct under "Local (GPU) screen.

### ISSUE THREAD

**Issue by Harika1702** [dotnet/machinelearning-modelbuilder#2674]
### GitHub Tags
#WCAG2.4.3; #A11ySev2; #A11yTCS; #A11yWCAG2.1; #DesktopApp; #Win11; #Visual Studio Client; #SH_ModelBuilder_Win32_May2023; #FTP; #Win32; #Benchmark; #Keyboard; #Tabstops; #A11yAuto; #Focus Order

### Environment Details
Application Name: Model Builder
Microsoft Visual Studio Enterprise 2022 (64-bit) - Current
Version 17.6.1
Microsoft Windows 11 Enterprise Build 22624.1755

### Repro Steps
1. Launch the Visual Studio with at least one of the .NET workloads.
2. Tab till enable ML.NET Model Builder (Tools -> Options -> Environment -> Preview Features) and select it.
3. Create a new .NET project or open an existing .NET project.
4. Open context menu (Right-click) of the project in the solution explorer.
5. Tab till Add and then tab till Machine Learning Model and press enter.
6. Model Builder dialog will open. Tab till Scenario tab.
7. Select Scenario screen will appear.
8. Tab till "Image Classification" and press enter.
9. Setup environment screen will appear, Tab till Local GPU tile and select it.
10. Now tab till the controls that are present under "Local (GPU) screen.
11. Now verify that Focus order is not correct under "Local (GPU) screen.

### Actual Result
Focus order is not correct under "Local (GPU) screen.
**Note:
We are facing similar issue throughout the application.**

### Expected Result
Focus order should be gone in proper order for "Local (GPU) Screen.
After Local (GPU) it should move to "How do I setup my local GPU for training?" then "Which GPU are compatible" then "Install CUDA" then "Install cuDNN".

### User Impact
Keyboard users will face difficulty to navigate to the controls present under "Local (GPU). 

### Attachment
![Bug_Focus order](https://github.com/dotnet/machinelearning-modelbuilder/assets/100270074/1db51cff-d745-407d-9e83-28633e0b183d)
<img width="960" alt="Bug_Tab stops" src="https://github.com/dotnet/machinelearning-modelbuilder/assets/100270074/24ef24a3-629b-417a-a06a-a3de0d56ceb0">

**Comment by zewditu** [comment]
@Harika1702, ready to validate by Model Builder version 16.17.0.2332602

**Comment by Samhitha66** [comment]
Regressed the issue in the below environment
Microsoft Visual Studio Enterprise 2022 (64-bit) - Current
Version 17.6.4
The issue is fixed. Hence closing the bug.
Please find the attachment for the reference.
#Regressed:06-30-23; #Closed;
![Focus order_Fixed](https://github.com/dotnet/machinelearning-modelbuilder/assets/99243079/910e2316-72b5-41eb-ab37-b8173fd31967)
